templates/cron.backup.j2

```diff
@@ -0,0 +1,15 @@
+# {{ ansible_managed }}
+#
+# Redis database backups
+#
+{% set dump = openio_redis_volume + "/dump.rdb" -%}
+{%- for backup in openio_redis_backups %}
+# {{ backup.name }}
+{% set backup_dir = openio_redis_volume + "/" + backup.name -%}
```

Comment:
should be `openio_redis_backup_directory` otherwise you are gonna backup to /mnt/ssd1/OPENIO/redis-0/hourly for example, which doesn't exist

Reply:
fixed